<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="main_area">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
       <item>
        <widget class="QWidget" name="upper_area" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
          <item>
           <widget class="QWidget" name="grid_diagram_area" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="maximumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="parameter_area" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1,5">
             <item>
              <widget class="QWidget" name="grid_diagram_data_area" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_gridSize">
                  <property name="text">
                   <string>网格图大小(X,Y)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spin_size_x">
                  <property name="minimum">
                   <number>2</number>
                  </property>
                  <property name="maximum">
                   <number>500</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spin_size_y">
                  <property name="minimum">
                   <number>2</number>
                  </property>
                  <property name="maximum">
                   <number>500</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="startpoint_area">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>起飞点位置(X,Y)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spin_start_x">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>500</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spin_start_y">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>500</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="endpoint_area">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_endPoint">
                  <property name="text">
                   <string>降落点位置(X,Y)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spin_end_x">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>500</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spin_end_y">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>500</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="obstacle_data_area" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,4">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QWidget" name="obstacle_num_area" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <widget class="QLabel" name="label_obstacleNum">
                     <property name="text">
                      <string>障碍物数量</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="spin_obstacle_num">
                     <property name="minimum">
                      <number>1</number>
                     </property>
                     <property name="maximum">
                      <number>10</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="obstacle_pos_area" native="true">
                  <layout class="QVBoxLayout" name="verticalLayout_10">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QFrame" name="frame">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::StyledPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="lower_area" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1">
          <item>
           <widget class="QWidget" name="priorities_area" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,5">
             <item>
              <widget class="QLabel" name="label_priority">
               <property name="text">
                <string>仿真首选项</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="priority_area" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QCheckBox" name="check_priority_1">
                  <property name="text">
                   <string>总路程</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="check_priority_2">
                  <property name="text">
                   <string>转弯数</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="check_priority_3">
                  <property name="text">
                   <string>总时间</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="check_priority_4">
                  <property name="text">
                   <string>综合</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="export_area" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,3,1">
             <item>
              <widget class="QLabel" name="label_export">
               <property name="text">
                <string>输出参数</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="export_priorities_area" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QCheckBox" name="check_export_1">
                  <property name="text">
                   <string>保存路径点（含代价）文件(.txt)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="check_export_2">
                  <property name="text">
                   <string>保存路径图(.png)</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="export_path_area" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_exportPath">
                  <property name="text">
                   <string>文件保存路径</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineedit_export_path">
                  <property name="minimumSize">
                   <size>
                    <width>300</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="search_button">
                  <property name="text">
                   <string>浏览</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="start_area" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <property name="topMargin">
              <number>11</number>
             </property>
             <item>
              <widget class="QLabel" name="label_warnmsg">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="start_button">
               <property name="text">
                <string>开始计算</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
